gatsby-config.js

```diff
@@ -64,6 +65,37 @@ module.exports = {
             'python',
           ],
         },
+        relatedResources: {
+          swiftype: {
+            resultsPath: `${__dirname}/src/data/swiftype-resources.json`,
+            engineKey: 'Ad9HfGjDw4GRkcmJjUut',
```

Comment:
Just to double-check, this is public and okay to commit, right?

Reply:
Yep! Its a client token specifically used for frontends to tell it which search engine to use.